src/templates/Gov/CreateProposal/index.tsx

```diff
@@ -0,0 +1,351 @@
+import Big from 'big.js'
+import { ethers } from 'ethers'
+import Tippy from '@tippyjs/react'
+import React, { FormEvent } from 'react'
+import { useConnectWallet, useSetChain } from '@web3-onboard/react'
+
+import { GovernorAlpha } from '@/constants/tokenAddresses'
+
+import useGov from '@/hooks/useGov'
+import { useAppDispatch } from '@/store/hooks'
+import { useVotingPower } from '@/hooks/query/useVotingPower'
+import { setModalAlertText } from '@/store/reducers/modalAlertText'
+
+import Button from '@/components/Button'
+import VotingPower from '@/components/VotingPower'
+import Breadcrumb from '@/components/Breadcrumb'
+import BreadcrumbItem from '@/components/Breadcrumb/BreadcrumbItem'
+import InputText from '@/components/Inputs/InputText'
+import MarkdownEditor from '@/components/MarkdownEditor'
+import TitleWithContador from './TitleWithContador'
```

Comment:
When you rename this component to `Counter`, remember to rename all imports.

```suggestion
import TitleWithCounter from './TitleWithCounter'
```